src/rpc/RPCHelpers.cpp

```diff
@@ -1329,4 +1332,51 @@ toJsonWithBinaryTx(data::TransactionAndMetadata const& txnPlusMeta, std::uint32_
     return obj;
 }
 
+std::expected<ripple::STArray, Status>
+fetchCredentials(
+    std::optional<boost::json::array> const& credID,
+    std::shared_ptr<BackendInterface> const& backend,
+    ripple::LedgerHeader const& info,
+    Context const& ctx
+)
+{
+    ripple::STArray authCreds;
+    if (credID.value().size() > ripple::maxCredentialsArraySize) {
```

Comment:
Do you still need this? The error message looks off too.

Reply:
Yea, the error is required for [this](url) test. The error message needs to be fixed though